### PR TITLE
Add conformance badge for k8s certification

### PR DESCRIPTION
**Pull request by lunarwhite**
*Issue #1421*

*Description of changes:*

* Add `certified-kubernetes-color.svg` to `docs/contents`
* Add badge for k8s certification

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,8 +3,8 @@
 ### 🚨🚨🚨 IMPORTANT INFORMATION ABOUT 1.20 SUPPORT 🚨🚨🚨
 
 EKS-D has discontinued support of Kubernetes v1.20. While there are no
-plans to removed EKS-D v1.20 images from the ECR, there will be no more 
-updates to v1.20. **Due to the increased security risk this poses, it is 
+plans to removed EKS-D v1.20 images from the ECR, there will be no more
+updates to v1.20. **Due to the increased security risk this poses, it is
 HIGHLY recommended that users of v1.20 update to a supported version (v1.21+)
 as soon as possible.**
```
Comment (TerryHowe) [inline README.md:9]: We aren't supporting 1.20 anymore, right. This should be removed?

File: README.md [hunk 2]
```diff
@@ -38,6 +38,8 @@ Full documentation for releases can be found on [https://distro.eks.amazonaws.co
 To receive notifications about new EKS-D releases, subscribe to the EKS-D updates SNS topic:
 `arn:aws:sns:us-east-1:379412251201:eks-distro-updates`
 
+[<img src="docs/contents/certified-kubernetes-color.svg" height=150>](https://github.com/cncf/k8s-conformance/pull/2252)
+
 ### Kubernetes 1-24
```
Comment (TerryHowe) [inline README.md:43]: We'd technically be certified for 1.24 and 1.23 if the conformance results have been submitted, but the older releases have already reached EOL has far as the community is concerned. This unversion badge may cause confusion. We should have a badge here though

File: README.md (hunk 1 above)
Comment (lunarwhite) [reply README.md:9]: Just trimmed two whitespaces here.

**Comment by lunarwhite** [comment]
Hi @TerryHowe , since the v1.24 badge is supported by cncf artwork, I've update this PR.

**Comment by markapruett** [comment]
@lunarwhite I was a little too quick to approve...Looks good, but please pull in the latest from `main` to ensure you're on the latest version of the README.md

**Comment by lunarwhite** [comment]
@markapruett Shall I open a new PR?

**Comment by markapruett** [comment]
@lunarwhite You should just be able to pull changes from `main` into your branch on your fork. The simplest way to do this is to go to your fork on GitHub and sync your fork with aws/eks-distro. When synced, you can pull the changes into your branch locally. But if you want to create a new branch and PR that's ok too.

**Comment by lunarwhite** [comment]
@markapruett Thanks for your guidance, I've synced the branch.

**Pull request merged**